src/binomial.jl

```diff
@@ -143,6 +148,13 @@ function ci_wald(x::BinomialTest, alpha::Float64=0.05)
     (quantile(Normal(μ, σ), alpha/2), quantile(Normal(μ, σ), 1-alpha/2))
 end
 
+# Wald interval with continuity correction
+function ci_waldcc(x::BinomialTest, alpha::Float64=0.05)
+    μ = x.x / x.n
+    σ = sqrt(μ*(1-μ)/x.n)
+    (quantile(Normal(μ, σ), alpha/2) - 0.5/x.n, quantile(Normal(μ, σ), 1-alpha/2) + 0.5/x.n)
```

Comment:
```suggestion
    lower, upper = ci_wald(x, alpha)
    correction = 1 // (2 * x.n)
    (lower - correction, lower + correction)
```
This reduces code duplication, plus it makes it more obvious at a glance that it's simply extending the Wald interval on both sides. Also, using a `Rational` in this way avoids promoting to `Float64` (in the off chance that `quantile` returns something other than `Float64`).